dmtestutils/login.py

```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint
+from dmutils.user import User
+from flask_login import login_user
+
+login_for_tests = Blueprint('login_for_tests', __name__)
+
+
+USERS = {
```

Comment:
We currently have _some_ API data fixtures in the apiclient in `api_stubs.py`. We were going to build this out a bit more, but maybe this repo is the right place to have them.

If that's the case, it would probably be good to move this to a separate file and generalise a bit.

Reply:
I think I need to take this ticket-by-ticket, or the thing will never land. Very happy for that API stubs thing to come in next, or as a new tech-debt ticket, or whatever - refactoring my one test that relies on check `USERS` won't be very hard.

Reply:
Cool, happy to bring the api stubs in myself if you want a hand getting this repo set up. 👍